pyproject.toml

```diff
@@ -1,8 +1,11 @@
 [project]
 name = "stactask"
 version = "0.6.0"
-authors = [{ name = "Matthew Hanson", email = "matt.a.hanson@gmail.com" }]
-maintainers = [{ name = "Pete Gadomski", email = "pete.gadomski@gmail.com" }]
+authors = [
+  { name = "Matthew Hanson", email = "matt.a.hanson@gmail.com" },
+  { name = "Pete Gadomski", email = "pete.gadomski@gmail.com" },
```

Comment:
I don't feel like I'm an author of this package so can you remove me here? TIA.

Reply:
Gotcha.